packages/core/tests/target.test.ts

```diff
@@ -0,0 +1,44 @@
+import {Target, TargetBuilder, TargetRuntime} from '..';
+import {
+  WebApp,
+  Package,
+  Service,
+  Project,
+  ProjectKind,
+  Runtime,
+  WorkspaceOptions,
+  Workspace,
+} from '../../model';
+import {
+  createTestWebAppWorkspace,
+  createTestServiceWorkspace,
+  createTestPackageWorkspace,
+  createTestComboWorkspace,
+} from '../../model/tests/utilities';
```

Comment:
Reaching in the other packages test utilities is generally discouraged. Ideally, this should live in its own testing packages like `@sewing-kit/testing` but for now, maybe we can move these into the root `tests/utilities`